### ISSUE TITLE
Builder won't connect to Path/Land marks

### ISSUE THREAD

**Issue by Xariyl** [AlexIIL/Buildcraft_1.8_Issues#69]
One cannot make chained-builds as the Builder won't connect to path markers or land marks.

![screenshot](http://i.imgur.com/F7xa2vL.png)

**Comment by AlexIIL** [comment]
This should now be fixed in pre2

**Comment by Xariyl** [comment]
Confirmed fixed as of buildcraft-7.2.0pre4

**Comment by AlexIIL** [comment]
from #70 

```
Ah sorry for not getting back to you: I was thinking of letting you close issues after 
you can confirm that the problem you had is fixed. This was I don't prematurely 
close an issue just because I no longer noticed it, as I may have missed something :)
```
